middleware/util/logging/configuration.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Shift Devices AG
```

Comment:
(Optional) Since I think we already have copyright headers like this in the repo, I won't mind it too much if we add more of them, but we should probably figure out a canonical Shift Cryptosecurity copyright header / software license once and for all (perhaps one exists?) and use it here and everywhere else.